### ISSUE TITLE
Problem using mk_prepare_ligand.py with  -v  in version 0.5.1

### ISSUE THREAD

**Issue by xavgit** [forlilab/Meeko#166]
Hi,
I receive the following  using the following:

$ mk_prepare_ligand.py -v  -i DB00116.sdf --merge_these_atom_types --rigid_macrocycles 
/usr/lib/python3/dist-packages/apport/report.py:13: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
  import fnmatch, glob, traceback, errno, sys, atexit, imp, stat
Traceback (most recent call last):
  File "/home/xxxx/.local/bin/mk_prepare_ligand.py", line 8, in <module>
    sys.exit(main())
  File "/home/xxxx/.local/lib/python3.8/site-packages/meeko/cli/mk_prepare_ligand.py", line 364, in main
    if args.verbose: preparator.show_setup()
AttributeError: 'MoleculePreparation' object has no attribute 'show_setup'

Without the -v there is not problem in the execution of the previous call to mk_prepare_ligand.py?

Any suggestion?

Thanks.

Saverio

**Comment by diogomart** [comment]
It's a bug, will be fixed in v0.6.0. You can change line 364 from
```python
            if args.verbose: preparator.show_setup()
```
to
```python
            if args.verbose: preparator.setup.show()
```
to fix it locally. Won't fix it for reactive docking if the `--reactive_smarts` matches more than once.

**Comment by diogomart** [comment]
It had been mostly fixed in the `develop` branch. Just added a minor fix in 05526d925d1269a444e210ca2564cfc6ff83b9ce #167
